expo-maps/android/src/main/java/expo/modules/maps/ExpoGoogleMapsModule.kt

```diff
@@ -18,6 +18,22 @@ class ExpoGoogleMapsModule : Module() {
       prop("mapType") { view: GoogleMapsView, mapType: MapType ->
         view.setMapType(mapType)
       }
+      prop("zoomControls") { view: GoogleMapsView, enabled: Boolean ->
+        view.setZoomControlButton(enabled)
+      }
+      prop("compass") { view: GoogleMapsView, enabled: Boolean ->
+        view.setCompassButton(enabled)
+      }
```

Comment:
What do you think about slightly different and in my opinion more clear naming convention ?
```suggestion
      prop("showCompassButton") { view: GoogleMapsView, enabled: Boolean ->
        view.setShowCompassButton(enabled)
      }
```